brion/detail/meshBinary.h

```diff
@@ -40,6 +40,19 @@ T get(const uint8_t* buffer, size_t& pos)
     return *val;
 }
 
+template <typename T>
+std::shared_ptr<std::vector<T>> readBuffer(const uint8_t* const ptr,
```

Comment:
According to CI there must be something wrong with this function. I think it's better to use const T* const ptr and the template parameter and do a static cast at call time.

Reply:
Yes, we talked offline and came to the same conclusion :)

Reply:
I added a cast inside the function instead to avoid the extra typing.